source/manual/email-notifications-how-they-work.html.md

```diff
@@ -70,7 +70,7 @@ https://email-alert-api-public.publishing.service.gov.uk.
 A courtesy copy of every email sent is available in [a Google Group][google-group].
 
 You can login to the Notify account by going to
-[https://www.notifications.service.gov.uk](). The login credentials are
+[https://www.notifications.service.gov.uk](https://www.notifications.service.gov.uk). The login credentials are
```

Comment:
```suggestion
<https://www.notifications.service.gov.uk>. The login credentials are
```

I think this is what the previous editor had tried to do, this format means you don't need to repeat the URL and the link text itself becomes the URL.

https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links